standard.mk

```
@@ -29,35 +29,41 @@ MAINPACKAGE=./cmd/manager
 GOENV=GOOS=linux GOARCH=amd64 CGO_ENABLED=0
 GOFLAGS=-gcflags="all=-trimpath=${GOPATH}" -asmflags="all=-trimpath=${GOPATH}"
 
+CONTAINER_ENGINE=`which podman 2>/dev/null || which docker 2>/dev/null`
+
 TESTTARGETS := $(shell go list -e ./... | egrep -v "/(vendor)/")
 # ex, -v
 TESTOPTS :=
 
 ALLOW_DIRTY_CHECKOUT?=false
 
+.PHONY: testx
+testx:
+	$(shell echo "$(CONTAINER_ENGINE)")
```

```suggestion
	@echo $(CONTAINER_ENGINE)
```

This should do the same, without a subshell.